### ISSUE TITLE
[paraglide-sveltekit] Version 0.13 breaks SvelteKit apps (Svelte 5)

### ISSUE THREAD

**Issue by yhnavein** [opral/inlang-paraglide-js#272]
Hi, I have today upgraded `@inlang/paraglide-sveltekit` from `0.12.1` to `0.13.0` and I am observing a set of errors in the DevTools console and my SvelteKit app is unusable.

Other relevant versions:
- `@sveltejs/kit`: `2.10.1`
- `svelte`: `5.10.1`

Screenshot of the problem in the DevTools:

![Zrzut ekranu 2024-12-11 o 18 09 24](https://github.com/user-attachments/assets/58e892c2-be26-4ab4-b17c-2e52d6141ac8)

Additionally: there is this new type error in the IDE:
![Zrzut ekranu 2024-12-11 o 18 10 22](https://github.com/user-attachments/assets/84eb4343-e4bc-4ee8-95a9-23ec5d11531f)

Full contents: 
```
Type '{ config: I18nConfig<"en" | "pl">; strategy: { getLanguageFromLocalisedPath: (localisedPath: string) => "en" | "pl" | undefined; getLocalisedPath: (canonicalPath: string, languageTag: "en" | "pl") => string; getCanonicalPath: (localisedPath: string, languageTag: "en" | "pl") => string; }; ... 4 more ...; route(trans...' is not assignable to type 'I18n<string>'.
  The types of 'config.runtime.setLanguageTag' are incompatible between these types.
    Type '(language_tag: "en" | "pl" | (() => "en" | "pl")) => void' is not assignable to type '(language_tag: string | (() => string)) => void'.
      Types of parameters 'language_tag' and 'language_tag' are incompatible.
        Type 'string | (() => string)' is not assignable to type '"en" | "pl" | (() => "en" | "pl")'.
          Type 'string' is not assignable to type '"en" | "pl" | (() => "en" | "pl")'.ts(2322)
```

Maybe those two are connected, or two separate issues. Bu I observe them together.

After downgrading `@inlang/paraglide-sveltekit` to `0.12.1` app works fine and there are no IDE errors reported.

My app is compliant with what is presented on: https://inlang.com/m/dxnzrydw/paraglide-sveltekit-i18n/manual-setup

**Comment by samuelstroschein** [comment]
are you using svelte 4 or 5?

**Comment by samuelstroschein** [comment]
0.13.1 is released. https://github.com/opral/inlang-paraglide-js/issues/262 is open again

**Comment by samuelstroschein** [comment]
Fixed in @inlang/paraglide-sveltekit: 0.15.0 via https://github.com/opral/monorepo/pull/3277
